### PR TITLE
Provide a e2e test for "withoutSync()" bug fix

### LINKED ISSUE #1733: Provide a e2e test for "withoutSync()" bug fix
Provide a e2e test for this fix #1719
For more details please check this [discussion](https://luigi-project.slack.com/archives/CMV4EME3G/p1605876323141200)

### PR DESCRIPTION
**Pull request by legteodav**
Provide a e2e test for "withoutSync()" bug fix

**Related issue(s)**
Fixes #1733

### CODE REVIEW

File: test/e2e-test-application/e2e/tests/1-angular/navigation.spec.js [hunk 1]
```diff
@@ -849,4 +849,227 @@ describe('Navigation', () => {
...
+        cy.wrap(result).find('a[data-testid="navigate-withoutSync-virtual-tree"]')
+        // checking if we have highlighted  menu item
+        cy.get('a[href="/projects/pr2/virtual-tree"]').should('exist').should('have.class','is-selected');
+
+
+        // checking if we have NOT highlighted  menu item
+        cy.get('a[href="/projects/pr2/settings"]').should('exist').not('.is-selected');
+
+        // CLICK ON navigate-withoutSync-virtual-tree
+        // linkManager().withoutSync().navigate('/projects/pr2/virtual-tree')
```
Comment (ndricimrr) [inline test/e2e-test-application/e2e/tests/1-angular/navigation.spec.js:1027]:
```suggestion
        // CLICK ON navigate-withoutSync-settings
        // linkManager().withoutSync().navigate('/projects/pr2/settings')
```

File: test/e2e-test-application/e2e/tests/1-angular/navigation.spec.js [hunk 2]
```diff
@@ -849,4 +849,227 @@ describe('Navigation', () => {
...
+      });
+
+    });
+
+    it('withSync -> it should change page', () => {
+      cy.expectPathToBe('/projects/pr2');
+      cy.getIframeBody().then(result => {
+        // Link on Main page PR2 exist
+        cy.wrap(result).contains(' with params: project to global settings and back');
+
```
Comment (ndricimrr) [inline test/e2e-test-application/e2e/tests/1-angular/navigation.spec.js:1046]:
```suggestion

```

File: test/e2e-test-application/e2e/tests/1-angular/navigation.spec.js [hunk 3]
```diff
@@ -849,4 +849,227 @@ describe('Navigation', () => {
...
+
+        // checking if we have NOT highlighted  menu item
+        cy.get('a[href="/projects/pr2/virtual-tree"]').should('exist').not('.is-selected');
+
```
Comment (ndricimrr) [inline test/e2e-test-application/e2e/tests/1-angular/navigation.spec.js:1050]:
```suggestion
```

File: test/e2e-test-application/e2e/tests/1-angular/navigation.spec.js [hunk 4]
```diff
@@ -849,4 +849,225 @@ describe('Navigation', () => {
...
+    });
+  });
+
+
+
+
+
+
 });
```
Comment (ndricimrr) [inline test/e2e-test-application/e2e/tests/1-angular/navigation.spec.js:1073]:
```suggestion
  });
});
```

File: test/e2e-test-application/e2e/tests/1-angular/navigation.spec.js [hunk 5]
```diff
@@ -849,4 +849,225 @@ describe('Navigation', () => {
...
+      cy.expectPathToBe('/projects/pr2');
+      cy.getIframeBody().then(result => {
+        // Link on Main page PR2 exist
+        cy.wrap(result).contains(' with params: project to global settings and back');
+
+        // checking if we have NOT highlighted  menu item
+        cy.get('a[href="/projects/pr2/virtual-tree"]').should('exist').not('.is-selected');
+
+        // CLICK ON navigate-withoutSync-virtual-tree
+        // linkManager().withoutSync().navigate('/projects/pr2/virtual-tree')
```
Comment (ndricimrr) [inline test/e2e-test-application/e2e/tests/1-angular/navigation.spec.js:1051]:
```suggestion
        // CLICK ON navigate-withSync-virtual-tree
        // linkManager().navigate('/projects/pr2/virtual-tree')
```

**Review by ndricimrr — CHANGES_REQUESTED** [review]

**Review by ndricimrr — APPROVED** [review]
LGTM 👍

**Comment by JohannesDoberer** [comment]
Waiting until #1760 is done.

**Review by JohannesDoberer — APPROVED** [review]

**Pull request merged**